### PR TITLE
Signal to suppliers that an Article is a proof (or had a proof) on fulfilment

### PR DESCRIPTION
**Pull request by drewwills**
Screenshot 2021-01-21 at 10 39 42

Screenshot 2021-01-25 at 12 31 15

### CODE REVIEW

File: src/main/java/com/mixam/mxjdf4/sdk/MxJdf4.java [hunk 1]
```diff
@@ -85,9 +85,6 @@ public void setJob(Job job) {
     }
 
     public Export getExport() {
-        if (export == null) {
-            export = new Export();
-        }
```
Comment (drewwills) [inline src/main/java/com/mixam/mxjdf4/sdk/MxJdf4.java:90]: ## ATTENTION EFFIE

@Effie -- Just like the `shipment` element, it seems to me that Mxjdf documents for ripped proof orders should not contain an `export` element.  I searched the code in `mixam-shop`, and I can find nothing that will break if these 3 lines are removed.

File: README.md [hunk 2]
```diff
@@ -238,14 +244,32 @@ The top level of the MxJdf document contains the following elements:
...
 |assumedPrintType|The expect print technology|NONE(0), <br>LITHO(1), <br>DIGITAL(2), <br>INKJET(3)|
+|jobType|Classification of print job|PRIMARY_PRINT(0), <br>RIPPED_PROOF(1), <br>HARD_COPY_PROOF(2)|
 
-# 12. Element: shipment
+# 12. Element: associatedProof
+```javascript
+"associatedProof": {
+    "proofJobNumber": "532809",
+    "proofJobType": 1,
+    "proofDate": 1579177108174
```
Comment (acolebourne) [inline README.md:260]: Why not human readable in standardised format?

**Review by acolebourne — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (drewwills) [reply README.md:260]: Very good question.

I used this approach to dates to match what's already there for `dateCreated` (top-level object).

File: src/main/java/com/mixam/mxjdf4/sdk/MxJdf4.java (hunk 1 above)
Comment (drewwills) [reply src/main/java/com/mixam/mxjdf4/sdk/MxJdf4.java:90]: Also price & currency.

**Pull request merged**